test/integration/patients/worklist/schedule.js

```diff
@@ -745,28 +727,33 @@ context('schedule page', function() {
   });
 
   specify('bulk edit', function() {
-    localStorage.setItem(`schedule_11111_${ workspaceOne.id }-${ STATE_VERSION }`, JSON.stringify({
-      clinicianId: '11111',
+    const testActions = _.times(20, index => {
+      return getAction({
+        relationships: {
+          owner: getRelationship(currentClinician),
+          state: getRelationship(index % 2 ? stateTodo : stateInProgress),
+        },
+      });
+    });
+
+    localStorage.setItem(`schedule_${ currentClinician.id }_${ workspaceOne.id }-${ STATE_VERSION }`, JSON.stringify({
+      clinicianId: currentClinician.id,
       customFilters: {},
       dateFilters: {
         dateType: 'due_date',
         selectedDate: null,
         selectedMonth: null,
         relativeDate: null,
       },
-      actionsSelected: { '1': true, '4444': true },
+      actionsSelected: {
+        [testActions[0].id]: true,
+        '4444': true,
```

Comment:
4444 is also some sort of uuid?

Reply:
Yes, I believe it's there to test the scenario where the user had it selected previously but the action no longer exists or was completed. And I switched to it be a `uuid()`.